packages/server/test/integration/http_requests_spec.coffee

```diff
@@ -43,6 +43,13 @@ removeWhitespace = (c) ->
   c = str.lines(c).join(" ")
   c
 
+trimString = (s) -> _.trim(s)
+nonEmpty = (s) -> !_.isEmpty(s)
+
+# removes leading whitespace and blank lines
+removeLeadingWhitespace = (text) ->
```

Comment:
Before it was hard to see what is going one in a blob of text

<img width="1409" alt="Screen Shot 2020-03-17 at 2 09 17 PM" src="https://user-images.githubusercontent.com/2212006/76888374-31cbb300-685a-11ea-9349-28676adb28bd.png">

Now it is pretty clear
<img width="1247" alt="Screen Shot 2020-03-17 at 2 16 21 PM" src="https://user-images.githubusercontent.com/2212006/76888394-3a23ee00-685a-11ea-925c-a9559b9b60c8.png">

Reply:
yeah good